src/primitives/polyline/styled.rs

```diff
@@ -246,6 +299,54 @@ mod tests {
         );
     }
 
+    #[test]
+    fn thick_stroke_translated() {
+        let mut display: MockDisplay<BinaryColor> = MockDisplay::new();
+
+        let styled = Polyline::new(&PATTERN)
+            .translate(Point::new(-4, -3))
+            .into_styled(PrimitiveStyle::with_stroke(BinaryColor::On, 4));
+
+        assert_eq!(
+            styled.bounding_box(),
+            Rectangle::new(Point::zero(), Size::new(28, 10))
+        );
+
+        styled.draw(&mut display).unwrap();
+
+        assert_eq!(
+            display,
+            MockDisplay::from_pattern(&[
+                "         #                  ",
+                "       #####            ##  ",
+                "      #######         ##### ",
+                "    ##########      ####### ",
+                "   #############  ##########",
+                " ######## ################# ",
+                "#######     #############   ",
+                " ####         #########     ",
+                "  #            ######       ",
+                "                 ##         ",
+            ])
+        );
+    }
+
+    #[test]
+    fn thick_stroke_points() {
+        let mut d1: MockDisplay<BinaryColor> = MockDisplay::new();
+        let mut d2: MockDisplay<BinaryColor> = MockDisplay::new();
```

Comment:
Another general question about coding style:

I've recently started to use this type code more often:
```suggestion
        let mut d1 = MockDisplay::<BinaryColor>::new();
        let mut d2 = MockDisplay::<BinaryColor>::new();
```
Should we use this in new code or do you prefer the other version?

Reply:
I'll change it to the new one, but I don't really mind either way :)

Reply:
Heh, turns out we don't need to specify the types in these tests at all